### DISCUSSION TITLE
Error creating classroom: Title format is invalid

### DISCUSSION THREAD

**Discussion by markkors** [community/community#124999] [GitHub Education]
### Hi everyone,

I'm using Github classroom now for several year with my student and from today when creating a new classroom i got an error: _Title format is invalid_

The name of my organisation is not to long (also the displayname). 

Is this a known problem and does anyone know how to solve this or what the cause is?

**Comment by hfoxwell** [comment]
I have the same issue. 
I have tried changing the display name of my organisation with no change to the behaviour. 
All my other classrooms are working as intended.

**Comment by cboettig** [comment]
I have recently encountered this issue as well.  I have been able to create and delete a GitHub Classroom on this organization each year for some 5+ years, so whatever is causing this seems to be recent...

**Reply by harrisonrob** [reply]
@cboettig, I am in similar situation, but only 2+ years.  I have support ticket open from last week, but no response from @githubsupport.

**Comment by JessRudder** [comment]
@markkors , @hfoxwell , @cboettig and @harrisonrob - apologies for the delay in addressing this. We believe we identified and fixed the issue. Could you please check and see if you're able to create new classrooms using the organizations that were failing before? 

If you're still having an issue (and you feel comfortable sharing) please let me know the name of the GitHub organization you're unable to use for a classroom and we'll troubleshoot some more.

**Reply by harrisonrob** [reply]
@JessRudder, thanks so much for addressing this issue!  I can confirm that I have created a new classroom in my organization that I was unable to create before!  I am happy to share with you details of my organization and classroom if it will help others resolve this issue.  Thanks!
